index.js

```diff
@@ -12,6 +12,8 @@ const debug = require('debug')('couchbase-driver')
  * @memberof Driver
  */
 const OPERATIONS = {
+  /** Insert operation */
+  INSERT: 'insert',
```

Comment:
this are exposed enums for the transform param in `atomic`, insert doesn't really make much sense here.